### ISSUE TITLE
How to get Redux Debugger?

### ISSUE THREAD

**Issue by lsps9150414** [react-community/create-react-native-app#187]
Hi, I'm trying to use this [React Native Debugger](https://github.com/jhen0409/react-native-debugger) in a fresh CRNA project by following [these simple steps](https://medium.com/react-native-development/develop-react-native-redux-applications-like-a-boss-with-this-tool-ec84bed7af8).

It works in an ejected CRNA project but not in one that's not ejected yet.
Is it possible to set it up without ejecting?

**Comment by jhen0409** [comment]
I just updated to latest version for try this, it works for me with [`debugger-open`](https://github.com/jhen0409/react-native-debugger/tree/master/patch), could you provide more details?

**Comment by lsps9150414** [comment]
@jhen0409 Here's what I did:

1. install React Native Debugger

2. install `react-native-debugger-open`
![2017-04-29 2 18 15](https://cloud.githubusercontent.com/assets/5022617/25541760/81a08016-2c82-11e7-8902-9a38d0dc36b3.png)

3. add `rndebugger-open` script:
```json
"scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "node node_modules/jest/bin/jest.js --watch",
    "postinstall": "rndebugger-open"
  },
```

4. edit my configStore:
```javascript
import { createStore, applyMiddleware, compose } from 'redux';
import reducers from '../reducers';

const middlewares = [];
let enhancer;

if (__DEV__) {
  enhancer = compose(
    applyMiddleware(...middlewares),
    global.reduxNativeDevTools ?
      global.reduxNativeDevTools(/*options*/) : noop => noop,
  );
}

export default function configureStore(preloadedState) {
  const store = createStore(reducers, preloadedState, enhancer);
  if (global.reduxNativeDevTools) {
    global.reduxNativeDevTools.updateStore(store);
  }
  return store;
}
```

**Comment by jhen0409** [comment]
Do you run `postinstall` script successfully?

Otherwise you can just run `open "rndebugger://set-debugger-loc?port=19001"` (on mac), it should work.

**Comment by lsps9150414** [comment]
Run `postinstall` successfully but nothing happens afterwards. Toggle on debugger still opens the web debugger.
![2017-04-30 4 12 44](https://cloud.githubusercontent.com/assets/5022617/25562698/e127c056-2dbf-11e7-818f-8ee3cd137002.png)

However running `open "rndebugger://set-debugger-loc?port=19001"` works!
How can I make `rndebugger-open` works as expected?

**Comment by jhen0409** [comment]
Sorry it does have the problem. 😔

It seems [`xdl`](https://github.com/expo/xdl/blob/master/src/Project.js#L674) have set `ELECTRON_RUN_AS_NODE` env, this is why it cannot open RNDebugger, I'm avoid the env in `rndebugger-open`. (Released as `v0.3.7`)

**Comment by lsps9150414** [comment]
Upgrading to v0.3.7 solves the problem! Thanks!
